website/docs/r/vapp_vm.html.markdown

```diff
@@ -69,6 +69,7 @@ The following arguments are supported:
 * `cpus` - (Optional) The number of virtual CPUs to allocate to the vApp
 * `initscript` (Optional) A script to be run only on initial boot
 * `network_name` - (Optional) Name of the network this VM should connect to
+* `vapp_network_name` - ((Optional; *v2.1*)) Name of the vApp network this VM should connect to
```

Comment:
Good addition. However, too many brackets and missing plus. Should be:
`(Optional; *v2.1+*)`

Reply:
Good spot, fixed that